src/main/scala/com/yotpo/metorikku/input/jdbc/JDBCInput.scala

```diff
@@ -0,0 +1,27 @@
+package com.yotpo.metorikku.input.jdbc
+
+import com.yotpo.metorikku.input.ReadableInput
+import com.yotpo.metorikku.session.Session.getSparkSession
+import org.apache.spark.sql.DataFrame
+
+case class JDBCInput(var name: String, connectionUrl: String, user: String,
+                     password: String, driver: Option[String], table: String,
+                     partitionColumn: Option[String], numberOfPartitions: Option[Int]) extends ReadableInput {
+  def read(): DataFrame = {
+    val url = connectionUrl + "?zeroDateTimeBehavior=convertToNull"
```

Comment:
whats this?

Reply:
It's something that's only related to mysql we can move it to the client's side

Reply:
https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-reference-configuration-properties.html